charts/seed-machines/charts/packet-machineclass/charts/utils-templates

```diff
@@ -0,0 +1 @@
+../../../../utils-templates
```

Comment:
Delete this file (the whole `packet-machineclass` (respectively, the whole `seed-machines) directory).